draft-ietf-rats-corim.md

```diff
@@ -1466,6 +1466,10 @@ CoRIM-based data structures define an external representation of Conceptual Mess
 Appraisal processing describes both mapping transformations and Verifier reconciliation ({{sec-verifier-rec}}).
 Non-CoRIM-based data structures require mapping transformation, but these are out of scope for this document.
 
+If a Verifier requires CoBOMs, then the CoRIM inputs indicated explicitly by the CoBOM are the only appropriate and required CoRIMs.
```

Comment:
Pending a new issue where it is stated that a Verifier requires a CoBOM that text needs to be revisited before taking this PR further

Reply:
@yogeshbdeshpande I don't understand this comment.  Could you please clarify?

Reply:
Thank you, but I still don't understand your point. L1469 takes no stance on who decides whether CoBOMs are used or not; it purely describes the mechanism. Who sets the boolean `use_cobom` on which the behaviour switches is out of scope for this paragraph.

Reply:
The confusion stems from the fact that:
`If a Verifier requires CoBOM`
this terminology is ambigous! as it dictates Verifier is the authority for deciding the need for CoBoM